sky/clouds/aws.py

```diff
@@ -110,9 +110,17 @@ def get_default_ami(cls, region_name: str, instance_type: str) -> str:
                 }
                 assert region_name in amis, region_name
                 return amis[region_name]
-        # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20220308
         # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:64,:Ubuntu%2020,:Deep%20Learning%20AMI%20GPU%20PyTorch # pylint: disable=line-too-long
-        # Nvidia driver: 510.47.03, CUDA Version: 11.6
+        # There is an leter version as shown below, but as other clouds do not support
+        # torch==1.13.0+cu117 we do not apply it to avoid frequent updates:
+        # Deep Learning AMI GPU PyTorch 1.12.1 (Ubuntu 20.04) 20221025
+        #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (supports torch==1.13.0+cu117)
+        #     'us-east-1': 'ami-0eb1f91977a3fcc1b'
+        #     'us-east-2': 'ami-0274a6db2e19b7cc6'
+        #     'us-west-1': 'ami-0fb299af41d32cfd3'
+        #     'us-west-2': 'ami-04ba15f9bd464eb20'
+        # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20220308
+        #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (not supports torch==1.13.0+cu117)
```

Comment:
```suggestion

        # Commented below are newer AMIs, but as other clouds do not support
        # torch==1.13.0+cu117 we do not use these AMIs to avoid frequent updates:
        # Deep Learning AMI GPU PyTorch 1.12.1 (Ubuntu 20.04) 20221025
        #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (supports torch==1.13.0+cu117)
        #     'us-east-1': 'ami-0eb1f91977a3fcc1b'
        #     'us-east-2': 'ami-0274a6db2e19b7cc6'
        #     'us-west-1': 'ami-0fb299af41d32cfd3'
        #     'us-west-2': 'ami-04ba15f9bd464eb20'
        #
        # Current AMIs:
        # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20220308
        #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (does not support torch==1.13.0+cu117)
```